src/reputation/views/bounty_view.py

```diff
@@ -439,27 +439,29 @@ def filter_queryset(self, queryset):
         bounty_types = self.request.query_params.getlist("bounty_type")
         hub_ids = self.request.query_params.getlist("hub_ids")
 
+        # Build filters
         applied_filters = Q()
 
         # Only return bounties within specific hubs
         if hub_ids:
+            print("hub_ids", hub_ids)
             applied_filters &= Q(unified_document__hubs__id__in=hub_ids)
 
         # ResearchHub foundation only filter
-        if "researchhub" in bounty_types:
+        if "RESEARCHHUB" in bounty_types:
```

Comment:
Should we also add `RESEARCHHUB` as a bounty type choice?

Reply:
"RESEARCHHUB" as a filter type makes sense in the context of the bounties page to the user

From an API perspective, it should not be stored as a type in the database because it refers to a bounty that is created by the researchhub community. It is not a "type" per se, but from the user perspective, it makes sense:

<img width="303" alt="image" src="https://github.com/user-attachments/assets/e474a8d5-6da4-4b0c-ad08-8aa6027e14e4">